.github/workflows/container.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Container image build
+on:
+  pull_request:
+    types: [ closed ]
```

Comment:
If I am not mistaken if we will use: `on_push: master` then this job will be executed also for the opened PR, which I didn't want to. I thought it would be better to build&push the docker image only for merged PRs.

Reply:
With this setup, only when push to main branch, and the branch will be protected in this repo as the operator.  

```
on:
  push:
    branches: [ main ]
```